Makefile

```diff
@@ -27,7 +27,7 @@ coveralls: test
 lint: bandit black flake8 isort mypy ## Lint the repo
 
 bandit:
-	pipenv run bandit -r lambdas
+	pipenv run bandit -r ppod.py
```

Comment:
I'm using a `.bandit` file in my next PR to skip the test files since this approach will get annoying fast as more files are added

Reply:
It's possible we may not need to add more files, and can just keep everything in the `ppod.py` module.